src/folio_migration_tools/migration_tasks/loans_migrator.py

```diff
@@ -81,15 +83,26 @@ def __init__(
         self.tenant_timezone = ZoneInfo(self.tenant_timezone_str)
         self.semi_valid_legacy_loans = []
         for file_def in task_configuration.open_loans_files:
-            with open(
-                self.folder_structure.legacy_records_folder / file_def.file_name,
-                "r",
-                encoding="utf-8",
-            ) as loans_file:
-
+            loans_file_path = self.folder_structure.legacy_records_folder / file_def.file_name
+            with open(loans_file_path, "r", encoding="utf-8") as loans_file:
+                total_rows, empty_rows, reader = MappingFileMapperBase._get_delimited_file_reader(
+                    loans_file, loans_file_path
+                )
+                logging.info("Source data file contains %d rows", total_rows)
+                logging.info("Source data file contains %d empty lines", empty_rows)
+                self.migration_report.set(
+                    Blurbs.GeneralStatistics,
+                    f"Total lines in {loans_file_path.name}",
+                    total_rows,
+                )
+                self.migration_report.set(
+                    Blurbs.GeneralStatistics,
+                    f"Empty lines in {loans_file_path.name}",
+                    empty_rows,
```

Comment:
The terms "rows" and "lines" appear to be used somewhat interchangeably in this section, which I think could confuse users of the report and code.

I wold suggest either picking one word (even if it's imprecise), or differentiating more consistently.